### DISCUSSION TITLE
How to make a theme default to application?

### DISCUSSION THREAD

**Discussion by sksdutra** [alejandroautalan/pygubu-designer#236] [General]
Hello, I'm sorry if this is a trivial question, but I can't find an answer to it.

After I finish designing, I click on "Preview -> ttk -> clam", which is a theme that I really like, I do all the procedures to run the UI, but when I run the app with Python, this theme does not remain.

I even installed ttkthemes and other themes, but I still don't know how to make them the default when I finish designing and running the application.

Anyone can help?

Thanks!

**Comment by jrezai** [comment]
Hi @sksdutra,
Preview -> ttk is mainly for quick previews.
To create and use custom ttk styles that will stay with your application, use Ttk styles in Edit -> Preferences -> Ttk Styles

More details are here:
https://github.com/alejandroautalan/pygubu-designer/wiki/Ttk-Styles-Support

Or you can look into the `theme_use()` method, which is part of Tkinter.
https://tkdocs.com/tutorial/styles.html

I hope that helps.

**Reply by sksdutra** [reply]
Hi, @jrezai 

I did it, thank you very much!

I only added the lines after load the TopLevel window:

```python
from tkinter import ttk
(...)
ttk.Style().theme_use('clam')
```
